src/frontend/src/components/Header/Header.jsx

```diff
@@ -9,11 +9,7 @@ function Header() {
   const theme = useTheme();
   const matches = useMediaQuery(theme.breakpoints.down('sm'));
 
-  return (
-    <>
-      {matches ? <MobileHeader className="mobile-header" /> : <DesktopHeader className="header" />}
-    </>
-  );
+  return <>{matches ? <MobileHeader /> : <DesktopHeader />}</>;
```

Comment:
Why have you dropped these classnames?

Reply:
I think that change was made in one of the PRs?
https://github.com/Seneca-CDOT/telescope/pull/1148

I think this branch needs to be rebased.

Reply:
Yes. These lines are from another PR.  #1148